src/assemble_workflow/bundle.py

```diff
@@ -39,6 +40,12 @@ def __init__(self, build_manifest, artifacts_dir, bundle_recorder):
         self.min_tarball_path = self._copy_component(self.min_tarball, "dist")
         self.__unpack_min_tarball(self.tmp_dir.name)
 
+    def install_core(self):
```

Comment:
open to suggestions on what to call this function.

Reply:
install_min